.travis.yml

```diff
@@ -29,8 +29,13 @@ after_success:
 
 deploy:
 - provider: script
-  skip_cleanup: true
+  cleanup: false
```

Comment:
May work, but I usually set `cleanup: true` which is an equivalent of `git stash`, basically cleaning the env. If you don't do that, the go releaser may fail to publish the release saying that git state is dirty.

Reply:
fixed